docs/how-to/index.rst

```diff
@@ -1,10 +1,62 @@
 How-to guides
 =============
 
-These how-to guides will cover key operations and processes in MicroCeph.
+Our *how-to* guides give directions on how perform key operations and processes in MicroCeph.
+
+*********************************************
+Installing and initialising MicroCeph cluster
+*********************************************
```

Comment:
Use these standards for headings https://canonical-documentation-with-sphinx-and-readthedocscom.readthedocs-hosted.com/style-guide/

Reply:
Okay, definitely much cleaner.